enhancements/_title-YYYY-MM-DD-#issue.md

```diff
@@ -0,0 +1,62 @@
+# Enhancement Proposal (EP) <issue#> - <YYYY-MM-DD> - <title>
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals/Pain](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+    - [User Experience](#user-experience)
+    - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    - [Risks and Mitigations](#risks-and-mitigations)
+    - [Alternatives](#alternatives)
+- [Future Improvements](#future-improvements)
+
+## Summary
+
+- Provide a one-paragraph description of the expected change here.
```

Comment:
Maybe these hints should be HTML comments (similar to the hints in the PR template) to make sure they don't show up in the rendered content in case someone doesn't delete them?

Reply:
This is a good question. I consciously left these comments in plain text to kind of force deleting them because I assume we do not want to see documents with comments in our repository.